stdlib/2and3/traceback.pyi

```diff
@@ -82,7 +88,7 @@ if sys.version_info >= (3, 5):
 
 
 if sys.version_info >= (3, 5):
-    class StackSummary:
+    class StackSummary(List):
```

Comment:
Wouldn't that defeat the example I put in the description? _PT is a tuple while the elements of the list are frame objects.